client/state/login/selectors.js

```diff
@@ -132,7 +126,7 @@ export const isTwoFactorAuthTypeSupported = ( state, type ) => {
  * @returns {boolean}         Whether a login request is in-progress.
  */
 export const isRequesting = ( state ) => {
-	return get( state, 'login.isRequesting', false );
+	return state.login.isRequesting ?? false;
```

Comment:
No need for fallback, always a valid boolean.

Reply:
Done.